pr-data.csv

```diff
@@ -4261,7 +4261,7 @@ https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-t
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-testlib,com.google.common.collect.testing.testers.MapToStringTester.testToString_formatting[setUpTearDown [collection size: several] reserialized [collection size: several]],ID,,,
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-testlib,com.google.common.collect.testing.testers.MapToStringTester.testToString_formatting[setUpTearDown [collection size: several]],ID,,,
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToString_formatting[synchronized ArrayListMultimap [collection size: several].asMap [collection size: several]],ID,,,
-https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToString_formatting[Synchronized.setMultimap [collection size: several] reserialized [collection size: several].asMap [collection size: several]],ID,,,
+https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-testlib,com.google.common.collect.testing.testers.MapToStringTester.testToString_formatting[Synchronized.setMultimap [collection size: several] reserialized [collection size: several].asMap [collection size: several]],ID,MovedOrRenamed,,https://github.com/google/guava/commit/a197d99b96d69ea75a2c4a15ecd8a730dd3270d6
```

Comment:
Please move this change to another PR because `MapToStringTester` is discussed in several PRs.

Please also squash your 4+ commits into one for this PR (and you also had 2 commits in a real PR).